lib/workers/repository/dependency-dashboard.spec.ts

```diff
@@ -144,6 +144,7 @@ describe('workers/repository/dependency-dashboard', () => {
     it('open or update Dependency Dashboard when all branches are closed and dependencyDashboardAutoclose is false', async () => {
       const branches: BranchConfig[] = [];
       config.dependencyDashboard = true;
+      config.dependencyDashboardHeader = 'This is a header';
       config.dependencyDashboardFooter = 'And this is a footer';
```

Comment:
Still missing some template usage. The result should also validated, so the template is replaced with right values.